docker/r10k/Dockerfile

```diff
@@ -35,6 +35,9 @@ LABEL org.label-schema.maintainer="Puppet Release Team <release@puppet.com>" \
 COPY docker/r10k/adduser.sh docker/r10k/docker-entrypoint.sh /
 COPY docker/r10k/docker-entrypoint.d /docker-entrypoint.d
 
+ARG PP_USER
```

Comment:
This should probably have a default value of `ARG PP_USER=puppet` so that a default user is defined.